config/purifier.php

```diff
@@ -0,0 +1,90 @@
+<?php
```

Comment:
Did you talk with anyone about adding this dependency here?

Reply:
not because I thought it was obvious

Reply:
What was obvious?

Reply:
adding config to a package that requires it